src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
 	$.widget("ui.igBaseEditor", {
 		options: {
 			/* type="string|number|null" Gets/Sets how the width of the control can be set.
```

Comment:
I don't think the option control "how" the width of the control is set. It controls the width itself.